3-enrich/scala-common-enrich/src/main/scala/com.snowplowanalytics.snowplow.enrich/common/outputs/BadRow.scala

```diff
@@ -56,7 +73,7 @@ case class BadRow(
    */
   def toJValue: JValue =
     ("line"           -> line) ~
-    ("errors"         -> errors.toList) ~
+    ("errors"         -> errors.toList.map(_.toString)) ~
```

Comment:
When did we decide to low-fi the errors from a well structured JSON down to a string? Remember we are primarily loading these errors into Elasticsearch which has great support for JSONs.